### DISCUSSION TITLE
[x] chat info shortcuts: name, health, ammo etc.

### DISCUSSION THREAD

**Discussion by u1ps** [etlegacy/etlegacy#1958] [Ideas]
What about bringing this system to legacy?

few examples:
Great shot [d] (last person who killed you)
Need a medic [s] HP left (amount of HP)

![1505669087_Screenshot2022-04-13080219 png a74e6a12baaa99d0ca51893a7cf850da](https://user-images.githubusercontent.com/99591943/163676258-1bffafee-769d-40a8-9bb7-5aa7c1bd48a6.png)

**Comment by x0rnn** [comment]
This could work with a lua script, the vsays at least are in kspree.lua already.
